nf_core/pipeline-template/modules/local/get_software_versions.nf

```diff
@@ -21,13 +21,23 @@ process GET_SOFTWARE_VERSIONS {
     path versions
 
     output:
-    path "software_versions.tsv"     , emit: tsv
-    path 'software_versions_mqc.yaml', emit: yaml
+    path "software_versions.yml"     , emit: yml
+    path 'software_versions_mqc.yaml', emit: mqc_yaml
 
     script: // This script is bundled with the pipeline, in {{ name }}/bin/
     """
-    echo $workflow.manifest.version > pipeline.version.txt
-    echo $workflow.nextflow.version > nextflow.version.txt
-    scrape_software_versions.py &> software_versions_mqc.yaml
+    cat - $versions <<-END_WORKFLOW_VERSION > software_versions.yml
```

Comment:
Do I get it right that this command concatenates the collated `versions.yml` and the HEREDOC?

Reply:
Yes.